app/migrations/Version20180703113033.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Migrations;
+
+use AppBundle\Entity\Donation;
+use AppBundle\Entity\Transaction;
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20180703113033 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql(
+            'UPDATE donations SET donations.status = :status WHERE donations.duration != 0 AND donations.subscription_ended_at IS NULL',
+            ['status' => Donation::STATUS_SUBSCRIPTION_IN_PROGRESS]
+        );
+        $this->addSql(
+            'UPDATE donations '.
```

Comment:
You can use heretic syntax instead of those concatenations and whitespace at end of lines:
```php
    $this->addSql(<<<EOSQL
UPDATE donations
JOIN
    (SELECT MAX(id) max_id, donation_id FROM donation_transactions GROUP BY donation_id) dt_max
...
EOSQL
    );
```